src/Python-2.0.1.xml

```diff
@@ -0,0 +1,284 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license licenseId="Python-2.0.1" name="Python Software
+   Foundation License 2.0.1" listVersionAdded="3.18">
+      <crossRefs>
+         <crossRef>https://www.python.org/download/releases/2.0.1/license/</crossRef>
+         <crossRef>https://docs.python.org/3/license.html</crossRef>
+         <crossRef>https://github.com/python/cpython/blob/main/LICENSE</crossRef>
+      </crossRefs>
+      <notes>
+         This is the stack of licenses used for Python version 1.6.1 onward.
```

Comment:
Just confirming -- this is for 1.6.1 onwards? If so, should the ID and name of the license be `Python-1.6.1` instead of `Python-2.0.1`? Apologies if this was already sorted out...

Reply:
that would be logical... I'm not entirely sure on the "used for version onwards" to be honest, I think I got that from some of the research done above.
But 2.0.1 seems to be the "name" used in some places and it seems to also have come after the license stack we have as 2.0
I think the license version number and python version number don't necessarily match up. (numbering seems to be a bit "loose" in this context overall)

Reply:
added better note on this